patches/minecraft/net/minecraft/world/entity/LivingEntity.java.patch

```diff
@@ -588,3 +604,17 @@
           }
        } else {
           return EquipmentSlot.HEAD;
+@@ -3287,5 +_,13 @@
+    }
+ 
+    public static record Fallsounds(SoundEvent f_196626_, SoundEvent f_196627_) {
++   }
++
++   @Override
++   public int getTotalExperienceValue(Player player){
++      if(!m_6149_() && !m_6124_())
```

Comment:
Can you not get the total experience value of another player? I'm wondering the additional condition that they must not always drop experience has to do with the total experience value they drop.

Reply:
You cannot get the total experience value of another player from the method, but the total levels is exposed, but not helpful for what the actual raw exp value is.

Reply:
Should be fine then.